deps/build.jl

```diff
@@ -6,33 +6,29 @@ using Compat
 function validate(name, handle)
     try
         fhandle = Libdl.dlsym(handle, :zmq_version)
-        major = Array(Cint,1)
-        minor = Array(Cint,1)
-        patch = Array(Cint,1)
+        major = Vector{Cint}(1)
+        minor = Vector{Cint}(1)
+        patch = Vector{Cint}(1)
         ccall(fhandle, Void, (Ptr{Cint}, Ptr{Cint}, Ptr{Cint}), major, minor, patch)
         return VersionNumber(major[1], minor[1], patch[1]) >= v"3"
     catch
         return false
     end
 end
 
-zmq = library_dependency("zmq", aliases = ["libzmq"], validate=validate)
+zmq = library_dependency("zmq", aliases = ["libzmq"], validate = validate)
 
-provides(AptGet,"libzmq3-dev",zmq)
+provides(Sources, URI("https://archive.org/download/zeromq_3.2.4/zeromq-3.2.4.tar.gz"), zmq)
+provides(BuildProcess, Autotools(libtarget = "src/.libs/libzmq." * BinDeps.shlib_ext), zmq)
 
-provides(Sources,URI("https://archive.org/download/zeromq_3.2.4/zeromq-3.2.4.tar.gz"),zmq)
-provides(BuildProcess,Autotools(libtarget = "src/.libs/libzmq."*BinDeps.shlib_ext),zmq)
-
-if is_windows()
+if is_linux()
+    provides(AptGet, "libzmq3-dev", zmq, os = :Linux)
```

Comment:
I don't know how it worked since the package doesn't provide the unversioned file. Maybe BinDeps looks for versioned dependencies when the unversioned one is missing. Anyway, better add the versioned name to `alias` like e.g. Cairo does.